sdk/communication/azure-communication-common/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.2.2 (Unreleased)
```

Comment:
pls change the version in this [line](https://github.com/Azure/azure-sdk-for-java/blob/45b9fc2084327ab65e3db0fa255905f5dd3b8bd2/eng/versioning/version_client.txt#L63) in the version_client.txt file as follow, it will fix ci analyze task failure:
`com.azure:azure-communication-common;1.2.1;1.2.2`